packages/@react-spectrum/table/test/Table.test.js

```diff
@@ -2394,6 +2432,13 @@ describe('TableView', function () {
         expect(onSelectionChange).not.toHaveBeenCalled();
         expect(onAction).toHaveBeenCalledTimes(1);
         expect(onAction).toHaveBeenCalledWith('Foo 5');
+
+        // Android TalkBack double tap test
```

Comment:
This seems like it should be in a separate test, not a second half of a single tap onAction test. Or rename this.
And should we have a test for a non-android double tap?
Should the android double tap test have both events be android events?

Reply:
I included it here because this test case is testing screen reader tap interactions. The `userEvent.click` done just before this section of code is simulating what VoiceOver will fire on the cell upon a double tap. This block of code simulates the special case of Android TalkBack double tap but IMO still falls under the scope of this test case since it is testing the same interaction.

> And should we have a test for a non-android double tap

The stuff above my code is testing the non-android double tap case

> Should the android double tap test have both events be android events

The code here simulates the pointer event fired upon double tapping w/ Android TalkBack double tap